doc/sphinx/language/coq-library.rst

```diff
@@ -1,1112 +1,988 @@
-  & $|$ & {\term} {\tt =} {\term} & ({\tt eq})\\
-  & $|$ & {\term} {\tt =} {\term} {\tt :>} {\specif} & ({\tt eq})
-\end{tabular}
-\end{centerframe}
-\caption{Syntax of formulas}
-\label{formulas-syntax}
-\end{figure}
-
-The basic library of {\Coq} comes with the definitions of standard
+notations fixed in the initial state are :
+
+================  ============  ===============
+Notation          Precedence    Associativity
+================  ============  ===============
+``_ -> _``        99            right
+``_ <-> _``       95            no
+``_ \/ _``        85            right
+``_ /\ _``        80            right
+``~ _``           75            right
+``_ = _``         70            no
+``_ = _ = _``     70            no
+``_ = _ :> _``    70            no
+``_ <> _``        70            no
+``_ <> _ :> _``   70            no
+``_ < _``         70            no
+``_ > _``         70            no
+``_ <= _``        70            no
+``_ >= _``        70            no
+``_ < _ < _``     70            no
+``_ < _ <= _``    70            no
+``_ <= _ < _``    70            no
+``_ <= _ <= _``   70            no
+``_ + _``         50            left
+``_ || _``        50            left
+``_ - _``         50            left
+``_ * _``         40            left
+``_      _``      40            left
+``_ / _``         40            left
+``- _``           35            right
+``/ _``           35            right
+``_ ^ _``         30            right
+================  ============  ===============
+
+Logic
+~~~~~
+
+The basic library of |Coq| comes with the definitions of standard
 (intuitionistic) logical connectives (they are defined as inductive
 constructions). They are equipped with an appealing syntax enriching the
-(subclass {\form}) of the syntactic class {\term}. The syntax
-extension is shown on Figure~\ref{formulas-syntax}.
-
-% The basic library of {\Coq} comes with the definitions of standard
-% (intuitionistic) logical connectives (they are defined as inductive
-% constructions). They are equipped with an appealing syntax enriching
-% the (subclass {\form}) of the syntactic class {\term}. The syntax
-% extension \footnote{This syntax is defined in module {\tt
-%     LogicSyntax}} is shown on Figure~\ref{formulas-syntax}.
- 
-\Rem Implication is not defined but primitive (it is a non-dependent
-product of a proposition over another proposition).  There is also a
-primitive universal quantification (it is a dependent product over a
-proposition). The primitive universal quantification allows both
-first-order and higher-order quantification.
-
-\subsubsection[Propositional Connectives]{Propositional Connectives\label{Connectives}
-\index{Connectives}}
+subclass `form` of the syntactic class `term`. The syntax of `form`
+is shown below:
+
+.. /!\ Please keep the blanks in the lines below, experimentally they produce
+   a nice last column. Or even better, find a proper way to do this!
```

Comment:
The rendering does not look good to me...

![screenshot-2018-4-3 the coq library coq 8 8 beta1 documentation](https://user-images.githubusercontent.com/1108325/38251413-acad8fda-3751-11e8-969a-9c83e743e00f.png)